Firestore/Example/Podfile

```diff
@@ -56,15 +56,17 @@ end
 # required.
 def configure_local_pods()
   # Firestore is always local; that's what's under development here.
-  pod 'FirebaseFirestore', :path => '../..'
+  pod 'FirebaseFirestore', :path => '../../'
```

Comment:
Why slash added here and below?

Shouldn't we use the same pattern?

Reply:
I see it was already mixed.

Reply:
Good catch, the slash likely came from me copying and pasting something. I've removed it, and I don't think it will affect CI.